package.json

```diff
@@ -142,7 +142,8 @@
     "typescript": "^4.9.4",
     "webpack": "^5.88.2",
     "webpack-retry-chunk-load-plugin": "^3.1.1",
-    "wrangler": "^3.4.0",
+    "//": "downgraded-because-vercel-og-incompatible",
```

Comment:
can you test if we can have multple "//" or if we need to add some sort of "// wrangler" here to avoid conflicts